### ISSUE TITLE
Sanitize output name when saving on disk instead of randomizing them

### ISSUE THREAD

**Issue by Ravencentric** [Jaded-Encoding-Thaumaturgy/vs-preview#123]
Output name was randomized in https://github.com/Jaded-Encoding-Thaumaturgy/vs-preview/commit/0736b46dee753798d085e9408033a18df66a2e10 after https://github.com/Jaded-Encoding-Thaumaturgy/vs-preview/issues/92.

My request is to partially revert that from randomize to instead santize the output name to make them path compatible. Randomized filenames are a pain if I just wanna upload the already generated screenshots manually after one thing or another interrupted the upload ([session expired when using slow.pics account integration](https://github.com/Jaded-Encoding-Thaumaturgy/vs-preview/assets/78981416/443e8e91-807e-4451-bd14-5874338c3f7a), vsp crashing, slow.pics choking, etc). Private trackers also have their own comparison thing in their forums and randomized filenames means I  have to first go through and rename them all again before cross uploading them.

TLDR: Randomized filenames are harder to work with if my use case is something other than letting VSP do everything
